robot-name/example.js

```diff
@@ -1,47 +1,27 @@
-const HOW_MANY_CHARS = 26;
-const HOW_MANY_NUMBERS = 1000;
-const CHAR_CODE_A = 'A'.charCodeAt(0);
+const alpha = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ',
+  usedNames ={};
 
-let generatedNames = {};
+const random = (max) => Math.floor(Math.random()*max);
 
-function generateName() {
-  let name = '';
-  name += generateRandomChar();
-  name += generateRandomChar();
-  name += pad(generateRandomNumber());
-
-  return getNotGeneratedName(name);
-}
-
-function generateRandomChar() {
-  const random = Math.floor(Math.random() * HOW_MANY_CHARS);
-  return String.fromCharCode(CHAR_CODE_A + random);
-}
-
-function pad(number) {
-  return ('000' + number).slice(-3);
-}
-
-function generateRandomNumber() {
-  return Math.floor(Math.random() * HOW_MANY_NUMBERS);
-}
-
-function getNotGeneratedName(name) {
-  if (generatedNames[name]) {
-    return generateName();
-  }
-
-  generatedNames[name] = true;
+const generateName = () => {
+  let name = alpha.charAt(random(26))
```

Comment:
Should we extract the magic number `26` to a constant calculated from `alpha`'s length?